docs/reference/commandline/stack_config.md

```diff
@@ -26,7 +26,7 @@ Options:
       --help            Print usage
 ```
 
-Displays the configuration of a stack.
+Display the configuration of a stack.
```

Comment:
yes, it is not a syntax error. However, it may be keep the same format among docker reference document.  It does not use this the imperative form. Do you agree?